core/math/color.cpp

```diff
@@ -428,43 +447,24 @@ Color Color::from_string(const String &p_string, const Color &p_default) {
 	}
 }
 
-String _to_hex(float p_val) {
-	int v = Math::round(p_val * 255);
-	v = CLAMP(v, 0, 255);
-	String ret;
-
-	for (int i = 0; i < 2; i++) {
-		char32_t c[2] = { 0, 0 };
-		int lv = v & 0xF;
-		if (lv < 10) {
-			c[0] = '0' + lv;
-		} else {
-			c[0] = 'a' + lv - 10;
-		}
-
-		v >>= 4;
-		String cs = (const char32_t *)c;
-		ret = cs + ret;
-	}
-
-	return ret;
+Color Color::from_hsv(float p_h, float p_s, float p_v, float p_alpha) {
```

Comment:
The only real change here is rename `p_a` to `p_alpha` and removing `const` qualifier. The rest is just moving stuff around to match order in header.